src/libstd/sys/windows/stdio.rs

```diff
@@ -1,17 +1,18 @@
 #![unstable(issue = "0", feature = "windows_stdio")]
 
 use crate::char::decode_utf16;
-use crate::cmp;
 use crate::io;
-use crate::ptr;
-use crate::str;
+#[cfg(not(target_os = "uwp"))]
```

Comment:
Sort of like the stack stuff, this file may be best entirely duplicated as `stdio_uwp.rs` if it doesn't support the console functions and only supports the bytes interface.